docs/src/explanations/dpi.md

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
can you please add this to the ToC on the [explanations/index.md?](https://raw.githubusercontent.com/chipsalliance/chisel/main/docs/src/explanations.md)?